### DISCUSSION TITLE
Accessing AMT

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#6021] [General]
Hi,
How can I access the AMT from the Mesh?
I am able to access via Mesh Command.
Thanks.

![AMT1](https://github.com/Ylianst/MeshCentral/assets/102313998/29ab2347-d332-448d-8c1e-200ed19f8a58)
![AMT2](https://github.com/Ylianst/MeshCentral/assets/102313998/491817c9-c11e-439e-9340-365cf3ea4330)

**Comment by si458** [comment]
I'm not sure what the red warning says, but it's either invalid credentials or no credentials
So you need to click the blue download icon and enter the devices amt login details

**Reply by ghost** [reply]
The red warning means that it trys to log in with the given credentials which are correct.
I'm using the same credentials in the Mesh Command and it works there.

**Reply by si458** [reply]
you are using Linux on the remote host, i think the is an issue with AMT and linux devices?
this might be why?
try going into the Console tab and typing these few command and see what outputs it has
`amt`
`amtevents`
`amtconfig`

**Reply by si458** [reply]
also try going into the `My Server Console` tab, and typing `amtpasswords` and check it has at least the correct password in for that device

**Reply by ghost** [reply]
amtpasswords gives nothing.

> amtmanager
No managed Intel AMT devices

>  amtstats
Total Intel AMT devices: 1
Un-provisionned: undefined
Provisionned: 1
Versions: 
  11.8.94: 1

> amt
{
  core-ver: 1
  OsHostname: "Latitude5480"
  Flags: 4
  Versions: {
    Flash: "11.8.94"
    Netstack: "11.8.94"
    AMTApps: "11.8.94"
    AMT: "11.8.94"
    Sku: "16392"
    VendorID: "8086"
    Build Number: "4494"
    Recovery Version: "11.8.94"
    Recovery Build Num: "4494"
    Legacy Mode: "False"
  }
  UUID: "4c4c4544-0057-5310-8034-b6c04f4a4d32"
  ProvisioningMode: 1
  ProvisioningState: 2
  net0: {
    enabled: 1
    dhcpEnabled: 1
    dhcpMode: "PASSIVE"
    mac: "D8:9E:F3:FC:C5:42"
    address: "192.168.178.25"
  }
  net1: {
    enabled: 0
    dhcpEnabled: 1
    dhcpMode: "PASSIVE"
    mac: "00:00:00:00:00:00"
    address: "0.0.0.0"
  }
}

> amtconfig
13:25:35, LMS tunnel start.
13:25:35, Checking Intel AMT state...
13:25:39, Intel AMT connected.
13:25:40, Done.
13:25:40, LMS tunnel closed.

> amtevents
13:11:25, LMS tunnel start.
13:11:25, Checking Intel AMT state...
13:11:29, Intel AMT connected.
13:11:30, Done.
13:11:30, LMS tunnel closed.

**Reply by si458** [reply]
Right so it's connected to amt no problem?
Not sure why the amtpasswords is blank?

**Reply by ghost** [reply]
I tried to restart and still the same.
Is there a way to build the DB again in the server? Maybe a DB renewal?

**Reply by si458** [reply]
you just delete the `meshcentral.db` inside `meshcentral-data` BUT THIS IS VERY DANGEROUS AS YOU WILL LOSE EVERYTHING!

is the devices AMT activated to a different server? as i think a device can only be managed by a single AMT device,
you would need to deactivate the AMT first before it will reactivate with your server

**Reply by ghost** [reply]
The AMT was activiated in the server itself which is Linux

**Reply by mtran-88** [reply]
What if it was on a different server previously?  What do I need to do to get it on the new server?  Thanks!

**Reply by ghost** [reply]
Hi just wanted to ask is there is any solution to the AMT Issue?
In the server console I see that there is one device and on the other hand there no devices shown in the amtmanager and the passwords are empty.

> amtmanager
No managed Intel AMT devices
> amtpasswords

> amtstatsTotal Intel AMT devices: 1
Un-provisionned: undefined
Provisionned: 1
Versions:   11.8.94: 1

**Reply by mtran-88** [reply]
Seriously, I am still trying to figure out how to get the button on the AMT tab to show up.  I wish there was a step-by-step guide on how to set it up with a troubleshooting section.  i.e. First, make sure you have AMT on the device.  Second, use this app to check, and add this. and that...no AMT button, check this; want to switch to ACM, do this using the following etc...  I thought it was going to be easy after looking at the guides, but I am still lost.  :o(

**Reply by si458** [reply]
We know the guides are poor and we are working on trying to update them but every device is different and behaves differently. Remember @Ylianst was let from Intel, so Intel AMT support is subpar now! And it's community support only, unless u want to pay for proper support.

**Reply by ghost** [reply]
I understand offcourse, and it's not critical as I can manage the AMT with the Commander and it would be a nice to have also in the Mesh.

**Reply by si458** [reply]
@quizhead I think ur issue is because the host in the picture is linux, and I remember an issue somewhere to do with meshcentral couldn't control with Linux machines correctly? If I had another amt device to test I would do! But I only have 2 and they are both windows and don't want to format them and confuse meshcentral

**Reply by mtran-88** [reply]
Hi Si458, I appreciate you trying to help.  It looks like you are the only one responding to the questions.  I am putting a how to guide for myself, and if I get all of the AMT set up properly, I can share that with the community when done.   Of course, that is assuming I can get all of the AMT stuff going.  ;O)  I can also help answers questions here with what I know so that you are not the only one.

**Reply by si458** [reply]
@mtran-88 that's perfectly fine! We do have some docs/videos for amt, but no step by step instructions! https://ylianst.github.io/MeshCentral/intelamt/

**Reply by mtran-88** [reply]
Hi, I am going through all of it again since I could not get the AMT set up on the second NUC.   I am hoping that going through it again would shed some light on the steps that is required for getting AMT to show up.
